daemon/src/process_manager.rs

```diff
@@ -163,14 +163,29 @@ impl Actor {
 
                 tracing::info!(%txid, "Commit transaction published");
             }
+            RolloverCompleted { dlc } => {
+                tracing::info!("Rollover complete, monitoring new txs and oracle events");
```

Comment:
I think we should split this into:

1. Logging here that rollover completed
2. Logging in `monitor` that we are starting to monitor a new DLC
3. Logging in `oracle` that we are starting to monitor a new attestation